### ISSUE TITLE
Failt to use proj4 with rails.

### ISSUE THREAD

**Issue by haroldofurtado** [rgeo/rgeo-proj4#6]
### System: MacOSX Catalina 10.15.6
**Instaled from Homebrew**
```
brew install proj
```
**When I run:**
```
projsync --system-directory

At least one of --list-files, --file, --source-id, --area-of-use, --bbox or --all must be specified.

usage: projsync 
          [--endpoint URL]
          [--local-geojson-file FILENAME]
          ([--user-writable-directory] | [--system-directory] | [--target-dir DIRNAME])
          [--bbox west_long,south_lat,east_long,north_lat]
          [--spatial-test contains|intersects]
          [--source-id ID] [--area-of-use NAME]
          [--file NAME]
          [--all] [--exclude-world-coverage]
          [--quiet] [--dry-run] [--list-files]
```
 **Gemfile**
```
gem "rgeo", "~> 2.1"
gem "rgeo-proj4", "~> 2.0"
```
**Ruby Version**
```
2.5.8
```
**Rails Version**
```
rails 5.2.4.2
```
**rails c**
```
pry(main)> RGeo::CoordSys::Proj4.supported?                                                                                                              
false
```

**Comment by LocoDelAssembly** [comment]
Potential duplicate of https://github.com/rgeo/rgeo-proj4/issues/4

Which proj version do you have installed (`$ proj -v`)?

**Comment by keithdoggett** [comment]
Fixed by #10
